raiden_contracts/deploy/etherscan_verify.py

```diff
@@ -256,8 +256,13 @@ def _verify_singleton_contract(
     )
 
 
-def guid_status(etherscan_api: str, guid: str) -> Dict:
-    data = {"guid": guid, "module": "contract", "action": "checkverifystatus"}
+def guid_status(etherscan_api: str, guid: str, apikey: str,) -> Dict:
```

Comment:
trailing comma

Reply:
Huh, how did that get trough black and the linters? I'll fix it.

Reply:
I've also seen this. I think black doesn't fix that for whatever reason.